ultravox/tools/ds_tool/chunked_dataset.py

```diff
@@ -0,0 +1,395 @@
+# mypy: ignore-errors
+import fnmatch
+import json
+import math
+import re
+import warnings
+from io import BytesIO
+from pathlib import Path
+from typing import Optional, Union
+
+from datasets import Dataset
+from datasets import config
+from datasets.data_files import sanitize_patterns
+from datasets.info import DatasetInfo
+from datasets.info import DatasetInfosDict
+from datasets.naming import _split_re
+from datasets.splits import SplitDict
+from datasets.splits import SplitInfo
+from datasets.utils import logging
+from datasets.utils.metadata import MetadataConfigs
+from datasets.utils.py_utils import asdict
+from datasets.utils.py_utils import glob_pattern_to_regex
+from datasets.utils.py_utils import string_to_dict
+from huggingface_hub import CommitInfo
+from huggingface_hub import CommitOperationAdd
+from huggingface_hub import CommitOperationDelete
+from huggingface_hub import DatasetCard
+from huggingface_hub import DatasetCardData
+from huggingface_hub import HfApi
+from huggingface_hub.hf_api import RepoFile
```

Comment:
nit: the import format doesn't match ours. For example, please use `import datasets` instead of `from datasets import Dataset`.